pulumitest/import_test.go

```diff
@@ -0,0 +1,17 @@
+package pulumitest_test
+
+import (
+	"testing"
+
+	"github.com/pulumi/providertest/pulumitest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestImport(t *testing.T) {
+	t.Parallel()
+	test := pulumitest.NewPulumiTest(t, "testdata/yaml_empty")
+
+	res := test.Import("random:index/randomString:RandomString", "str", "importedString", "")
+
+	require.Contains(t, res.Stdout, "type: random:RandomString")
```

Comment:
Seems a bit brittle to rely on particular output from pulumi. Can we assert on the stack contents instead?

Reply:
This asserts on the generated code.

I've also added an assert for the state.